src/CaptureClient/include/CaptureClient/CaptureClient.h

```diff
@@ -78,7 +78,7 @@ class CaptureClient {
       const absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo>& selected_functions,
       const TracepointInfoSet& selected_tracepoints, double samples_per_second,
       orbit_grpc_protos::UnwindingMethod unwinding_method, bool collect_scheduling_info,
-      bool collect_thread_state, bool enable_introspection,
+      bool collect_thread_state, bool enable_introspection, bool enable_api,
```

Comment:
Careful that the order of `enable_api` and `enable_introspection` is different in the .cpp.

Reply:
yikes, thanks.